src/lib/datepicker/calendar-header.html

```diff
@@ -1,9 +1,11 @@
 <div class="mat-calendar-header">
   <div class="mat-calendar-controls">
     <button mat-button type="button" class="mat-calendar-period-button"
-            (click)="currentPeriodClicked()" [attr.aria-label]="periodButtonLabel">
+            (click)="currentPeriodClicked()" [attr.aria-label]="periodButtonLabel"
+            cdkAriaLive="polite">
```

Comment:
Just noticed this now, but we should make `cdkAriaLive` default to `polite` in a separate PR

Reply:
created #11618 so we don't forget